<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deleteObjectDialog</class>
 <widget class="QDialog" name="deleteObjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Удаление объекта</string>
  </property>
  <widget class="QPushButton" name="deleteObjectOnlyButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Удалить только объект.
Статистика и другие данные не будут удалены.</string>
   </property>
   <property name="text">
    <string>Удалить объект</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Имя</string>
   </property>
   <widget class="QLineEdit" name="nameLine">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Адрес</string>
   </property>
   <widget class="QLineEdit" name="addressLine">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Тип</string>
   </property>
   <widget class="QLineEdit" name="typeLine">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>171</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Описание</string>
   </property>
   <widget class="QPlainTextEdit" name="descriptionText">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="deleteAllButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Удалить объект и все связанные с ним данные.</string>
   </property>
   <property name="text">
    <string>Удалить всё</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
